Monika After Story/game/script-greetings.rpy

```diff
@@ -2615,20 +2615,83 @@ label greeting_back_from_work:
         m 1hua "Oh, welcome back, [player]!"
         menu:
             m "Did you have a good day at work today?"
+            "I had a {i}really{/i} good day today.":
```

Comment:
Same as the other PR, could you please reorder these as Yes, better yes, No, worse no